fair_mango/dataset/dataset.py

```diff
@@ -3,11 +3,10 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
-import seaborn as sns
 
 
-def check_column_in_df(df: pd.DataFrame, columns: Sequence) -> None:
-    """Validate the columns existance in the dataset.
+def check_column_existence_in_df(df: pd.DataFrame, columns: Sequence) -> None:
+    """validate the columns existence in the dataset
```

Comment:
Yup, I missed that. Fixed

Reply:
I faced some issue committing because of mypy complaints. Can you check this [commit](https://github.com/datategy/Fair-Mango/pull/21/commits/8af0c99ffaa26ffd013feeba669814fff95ea627). The error message of mypy is:
```
tests/dataset/test_dataset.py:52: error: "check_column_existence_in_df" does not return a value (it only ever returns None)  [func-returns-value]
tests/dataset/test_dataset.py:77: error: "check_real_and_predicted_target_match" does not return a value (it only ever returns None)  [func-returns-value]
tests/dataset/test_dataset.py:389: error: "validate_columns" does not return a value (it only ever returns None)  [func-returns-value]
```

Reply:
yes, `check_column_existence_in_df` does not have a return statement, so `assert check_column_existence_in_df(df, columns) is expected_result` is equivalent to `assert None is expected_result`

Reply:
which is a reasonable thing to do, don't you agree? By removing that basically I am running the tests that raise an error only.

Reply:
your function always returns None, what would you want to test?